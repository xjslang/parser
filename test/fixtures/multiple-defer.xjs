function test() {
  console.log('start')
  defer {
    console.log('defer 1')
  }
  console.log('middle')
  defer {
    console.log('defer 2')
  }
  console.log('end')
}
test()
