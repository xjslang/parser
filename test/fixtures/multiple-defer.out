start
middle
end
defer 2
defer 1
